src/components/configuration/sdk-config-docs/cordova.js

```diff
@@ -101,6 +101,25 @@ const framework = docsPrefix => {
             ]
           }
         ]
+      },
+      security: {
+        serviceLogoUrl: '/img/security.svg',
+        serviceName: 'Security',
+        serviceDescription:
+          'Installs a security service which gives developers the ability to control their apps from the Mobile Security Console.',
+        setupText: 'Security SDK setup',
+        docsLink: `${docsPrefix}/security.html`,
```

Comment:
don't think this will work

Reply:
Can you be more specific about what you don't think will work. Its unclear if your talking about the `setupText` or `docsLink`

Reply:
@finp We are going to address your concerns with the `docsLink` string in a separate PR. It affects more than just the MSS